api/types.go

```diff
@@ -289,11 +289,11 @@ type ListResponse struct {
 type ModelResponse struct {
 	Name       string       `json:"name"`
 	Model      string       `json:"model"`
-	ModifiedAt time.Time    `json:"modified_at,omitempty"`
+	ModifiedAt *time.Time   `json:"modified_at,omitempty"`
```

Comment:
pointers in api struct isn't super ergonomic. a nicer solution could be to extend time.Time with a custom `json.Marshaler` and `json.Unmarshaler` similar to Duration